lib/matplotlib/tests/test_legend.py

```diff
@@ -1435,3 +1435,50 @@ def test_legend_text():
         leg_bboxes.append(
             leg.get_window_extent().transformed(ax.transAxes.inverted()))
     assert_allclose(leg_bboxes[1].bounds, leg_bboxes[0].bounds)
+
+
+def test_boxplot_labels():
```

Comment:
This got somehow messed up: The function was moved in #27901. It should not be added here again.